avm/res/app/managed-environment/main.bicep

```diff
@@ -334,6 +349,23 @@ type certificateKeyVaultPropertiesType = {
   keyVaultUrl: string
 }?
 
+type certificateObjectType = {
+  @description('Optional. The name of the certificate.')
+  name: string?
+
+  @description('Optional. The type of the certificate.')
+  certificateType: ('ServerSSLCertificate' | 'ImagePullTrustedCA')?
+
+  @description('Optional. The value of the certificate. PFX or PEM blob.')
+  certificateValue: string?
+
+  @description('Optional. The password of the certificate.')
+  certificatePassword: string?
+
+  @description('Optional. A key vault reference.')
+  certificateKeyVaultProperties: certificateKeyVaultPropertiesType
```

Comment:
Note, the `certificateKeyVaultPropertiesType` can be imported from the `certificates` child module through the `export()` statement

Reply:
```bicep
import { certificateKeyVaultPropertiesType } from 'certificates/main.bicep'
```